modules/imgimport/include/csvreading.h

```diff
@@ -0,0 +1,22 @@
+#ifndef CSV_READING_H_INCLUDED
+#define CSV_READING_H_INCLUDED
+
+#include "metadata.h";
+using namespace std;
```

Comment:
I think I forgot to note this before, but including using namespace (or even using by itself) in a header is not recommended, since now every file that directly or indirectly  includes this file (i.e, since imageimport.h includes csvreading.h, then this applies to anything that includes imageimport.h) is forced to include this line. 
It's generally preferable to explicitly state, for example, std::vector in a header to avoid this (though this file isn't actually using anything from the std namespace anymore, but imgimport.h is).